101-lab/lab-notes.md

```diff
@@ -0,0 +1,130 @@
+
+## Prep
+
+- create a rocketchat channel of the form `#ocp101-YYYY-MM-DD` with date being the date of the training session (the day before kickoff)
+- clean up the `d8f105` namespaces (let's make a script for this?)
+- post and pin:
+
+```
+Useful links:
+
+OCP tools namespace: https://console.apps.silver.devops.gov.bc.ca/topology/ns/d8f105-tools?view=graph
+OCP dev namespace: https://console.apps.silver.devops.gov.bc.ca/topology/ns/d8f105-dev?view=graph
+lab material: https://ocp101-labs-d8f105-tools.apps.silver.devops.gov.bc.ca/
+lab repo if you’d like to host the material later on: https://github.com/BCDevOps/devops-platform-workshops/tree/master/101-lab
+Just Ask, to request an invite to our GitHub orgs: https://just-ask-web-bdec76-prod.apps.silver.devops.gov.bc.ca/
+```
+
+## Kick-Off
+
+The day after the training, the lab has a kick-off meeting, during which we set up all the necessary access, provide an introduction to Openshift and the lab itself, and answer questions.
+
+- General Lab Intro Information
+    - lab is self-directed over about 2 weeks
+    - 3 meetings
+    - if you aren't finished in the next 2 weeks, let us know and we'll do what we can to keep your access to the namespaces a little longer
+    - point out and show off links to the lab content and content repo
+- RocketChat and GitHub Access
+    - Everyone sign into RocketChat and join the channel (show them how?)
+        - Mention the "new user sign-ups disabled" message, and to ignore it and click login anyway.
+    - Post your GitHub username in RocketChat
+    - Is everyone a member of the BCGov orgs in GitHub? If not, use the Just Ask link.
+- OpenShift Tour (while we set up everyone's RocketChat and GitHub Access)
+    - Tour of the OpenShift console
+    - How to download the CLI
+    - How to log in to the CLI
+    - Show the lab namespaces
+        - Talk about the namespace name (how the lab will sometimes use `ocp101-tools` or `[-tools]` and what to use instead)
+        - Mention that the attendees don't have access yet, not to worry!
+- Lab Step 1: Access Control
+    - Talk about the "team" format of the lab (aka most of you are devs, but we need a few devops specialists to be your admins)
+    - Get volunteer admins (at least two)
+    - Walk through the process of adding people to the namespace
+        - Mention the need to include `@github` and the fact that names are lowercase only
+    - Get the admins to work through the rest of the list
+- Tips and Tricks
+    - show off `oc explain` and `-h` flag
+    - mention the deployment vs deploymentconfig issue (there is a step where you're asked to issue an `oc rollout` command which won't work - you can skip this step)
+- Questions?
+
+## Mid-Lab Check-In
+
+- Questions? 
+- Talk about Infrastructure As Code
+    - TODO: Maybe let's build an example set of yaml files that produces everything from the lab? Then the teams can look at this and use it as an example!
```

Comment:
shall we use the lab material deployment itself then? to go through the process:
- oc get the objects
- edit the file
- delete the deployment and oc apply to show it pops up again
- then edit it to templates
- oc create a new instance